doc/Command_Index_User.tex

```diff
@@ -237,13 +237,13 @@ \subsubsection{Fine Tuning Dimensions}
 In addition to providing control over the overall size of your scores, Gregorio\TeX\ allows you to fine tune the spacings around and between the various elements using the following functions.
 
 \macroname{\textbackslash grecreatedim}{\{\#1\}\{\#2\}\{\#3\}}{gregoriotex-spaces.tex}
-Macro to create one of Gregorio\TeX’s distances.  Used to initialize distances in a space configuration file.
+Macro to create one of Gregorio\TeX’s distances.  Used to initialize distances in a space configuration file.  See \textit{gsp-default.tex} for an example of such a file.
```

Comment:
gsp-default.tex is not just an example of such a file.  It is the file used by default.